PhoneFormat.js

```diff
@@ -1,524 +1,441 @@
-/*
-
-Compiled Wednesday July 30, 2014 at 2:15pm America/New_York
-
-Version: libphonenumber r680
-
-------------------------------------------------------------------------
-
-
- Copyright (C) Alan Beebe (alan.beebe@gmail.com).
-
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-
- http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
-
-
-
-Usage...
-
-This is the same type of code used by cell phones when you enter
-a phone number into your dialer app.  Your phone already knows
-what country you are a subscriber in, so it assumes you are entering
-a local number, unless of course you prefix the number with a +, or
-in the USA you could also prefix the number with 011 to indicate you
-wish to dial internationally.  This code functions the same way.
-
-Lets assume your in the United States and you enter the following
-phone number: 8646978257
-
-formatE164("US", "8646978257");
- Returns: +18646978257
+Phone = {};
 
-countryForE164Number("US", "+18646978257");
- Returns: US
+// Get the user's current country based on the user's ip address
+Phone._getCountryFromIP = function (callback) {
+  var country = localStorage.getItem('current_country');
```

Comment:
Prefix this to not run into other people's local storage. Like `pf.country_code`